pkg/utils/impression.go

```diff
@@ -46,21 +46,21 @@ func CreateImpressionForPush(vwoInstance schema.VwoInstance, tagKey, tagValue, u
 	parameters.Add(tagKey, tagValue)
 	impression.U = parameters.Encode()
 
-	message := fmt.Sprintf(constants.DebugMessageImpressionForPush, vwoInstance.API, impression)
+	message := fmt.Sprintf(constants.DebugMessageImpressionForPush, vwoInstance.API, impression.AccountID, impression.UID, impression.SID, impression.URL, impression.U)
```

Comment:
no need to send seperately.

Reply:
Will %+v log all values along with the names , Like Key : value pair ?